Src/FluentAssertions/ObjectAssertionsExtensions.cs

```diff
@@ -9,6 +8,7 @@
 using FluentAssertions.Execution;
 using FluentAssertions.Primitives;
 
+#pragma warning disable CA1062 // Validate arguments of public methods
```

Comment:
Why?

Reply:
I was unsure how to handle CA1062 warnings.
CA1062 should be at least treated as warning, as methods of public methods *should* be validated, but it needs to be fixed in separate commit/PR as it currently produces 350+ warnings.
So I've made CA1062 a suggestion instead and removed this `#pragma`